net/device/MessageHelpers.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Microsoft.Azure.Devices.Client;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.Iot.Thief.Device
+{
+    internal static class MessageHelpers
+    {
+        /// <summary>
+        /// Reported max of entire payload is 256k, but we'll hold back 1 k as fudge factor for things we can't calculate.
+        /// </summary>
+        public const int MaxMessagePayloadBytes = (256 * 1024) - 1024;
+
+        private const int Iso8601Length = 24;
+
+        public static int GetMessagePayloadSize(Message message)
```

Comment:
I know this is muddy because of iothub mixing byte count and character count, but here you use the suffix "Size" and above you use the suffix "Bytes" in MaxMessagePayloadBytes.  This leads to 2 concerns:
1. Is the limit really 255K bytes or is is really 255k characters? 
2. If "character" is the unit of measurement that we're using, that should be explicit, maybe in the function name.
I know that C# is safer than C, but mixing "count of bytes" and "count of characters" is a common mistake and a great way to introduce buffer overruns.

Reply:
The mix of characters and bytes isn't my mistake, but I am staying true to the mistake that was made.